syncer/syncer_test.go

```diff
@@ -326,55 +415,44 @@ func (s *testSyncerSuite) TestSelectTable(c *C) {
 		}
 		i++
 	}
-	s.catchUpBinlog()
 }
 
 func (s *testSyncerSuite) TestIgnoreDB(c *C) {
-	s.resetMaster()
-	s.resetBinlogSyncer()
-
 	s.cfg.BWList = &filter.Rules{
 		IgnoreDBs: []string{"~^b.*", "s1", "stest"},
 	}
 
-	sqls := []string{
-		"create database s1",
-		"drop database s1",
-		"create database s2",
-		"drop database s2",
-		"create database btest",
-		"drop database btest",
-		"create database b1",
-		"drop database b1",
-		"create database stest",
-		"drop database stest",
-		"create database st",
-		"drop database st",
-	}
-	res := []bool{true, true, false, false, true, true, true, true, true, true, false, false}
+	s.resetEventsGenerator(c)
+	events := make([]mockBinlogEvent, 0, 12)
```

Comment:
```
events := mockBinlogEvents{
    mockBinlogEvent{typ: DBCreate, args: []interface{}{"s1"}},
    mockBinlogEvent{typ: DBDrop, args: []interface{}{"s1"}},
    // ...
}
```
this will be simpler

Reply:
Done